examples/pipelines/windows-customize/README.md

```diff
@@ -1,11 +1,16 @@
-# Windows 10 Customize Pipeline
+# Windows Customize Pipeline
 
-This pipeline clones the DataVolume of a basic and generalized Windows 10 installation and runs arbitrary customization
-commands through an unattend.xml after startup  of the VM. As an example a ConfigMap which installs Microsoft SQL
-Server Express and generalizes the VM after is included (`windows10-sqlserver`). For basic setup after the first start
-of a customized VM an example unattend.xml is included in the pipeline's ConfigMap `windows10-unattend`.
+This pipeline clones the DataVolume of a basic and generalized Windows 10/11/2k22 installation and runs arbitrary customization
+commands through an unattend.xml after startup of the VM. As an example a ConfigMap which installs Microsoft SQL
+Server Express and generalizes the VM after is included (`windows-sqlserver`) or a ConfigMap that install VSCode (`windows-vs-code`). 
+For basic setup after the first start of a customized VM an example unattend.xml is included in the pipeline's ConfigMap `windows10-unattend`, 
```

Comment:
Newline? Also might make sense to drop the `windows10-unattend` and `windows11-unattend` ConfigMaps from TTO and move them here, as they are only used as examples?